cli/util/logger.rs

```diff
@@ -38,6 +38,9 @@ pub fn init(maybe_level: Option<log::Level>) {
   )
   // https://github.com/denoland/deno/issues/6641
   .filter_module("rustyline", log::LevelFilter::Off)
+  .filter_module("swc_ecma_codegen", log::LevelFilter::Off)
+  .filter_module("swc_ecma_transforms_optimization", log::LevelFilter::Off)
+  .filter_module("tracing", log::LevelFilter::Off)
```

Comment:
@mmastrac could you give me a hand here? Updating hyper and related crates caused some tracing/log outputs to be displayed and I had to resort to doing this filtering, but it seems brittle (I might have missed something, even though the tests pass). I'm having trouble which crate caused this and it feels like a bug in one of these crates.